bpython/cli.py

```diff
@@ -160,6 +161,7 @@ def __init__(self, interface):
         self.encoding = getpreferredencoding()
         self.interface = interface
         self.buffer = list()
+        self.errors = "strict"
```

Comment:
Why? Does this fix a bug? If so, this should be a separate commit in a separate pull request.